setup.cfg

```diff
@@ -52,7 +52,6 @@ install_requires =
     flatten_dict
     openpyxl
     pydantic
-    python>=3.8
```

Comment:
You might need `python">="3.8`

Reply:
Thanks! It ended up being that there is a separate flag to specify the python version in the `setup.cfg` file that I just missed seeing 😅 haha